plugin/characterize.vim

```diff
@@ -30,12 +31,13 @@ function! s:info(char) abort
     for emoji in characterize#emojis(nr)
       let out .= ', '.emoji
     endfor
-    let entity = characterize#html_entity(nr)
-    if !empty(entity)
-      let out .= ', '.entity
-    endif
     call add(outs, out)
   endwhile
+  let entity = characterize#html_entity(matchstr(getline('.')[col('.')-1:-1],'.'))
```

Comment:
Use `a:char` here.

Reply:
Done, tested, no issues. (And, incidentally, my temporarily commented line 17, which I'd not noticed and should have been removed, has been deleted.)